sites/vehicleservicepros.com/server/templates/website-section/videos.marko

```diff
@@ -0,0 +1,142 @@
+    </marko-web-resolve-page>
+  </@above-container>
+  <@page>
+    <marko-web-gam-display-ad id="gpt-ad-lb1" modifiers=["top-of-page"] />
+
+    <marko-web-page-wrapper class="mb-block">
+      <@section>
+        <div class="row">
+          <div class="col">
+            <default-theme-breadcrumbs-with-home>
+              <@item>${title}</@item>
+            </default-theme-breadcrumbs-with-home>
+            <h1 class="page-wrapper__title">${title}</h1>
+          </div>
+        </div>
+      </@section>
+    </marko-web-page-wrapper>
+
+    <div class="row">
+      <div class="col">
+        <marko-web-query|{ nodes }|
+          name="all-published-content"
+          params={ contentTypes: ["Video"], limit: 5, queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes with-teaser=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">UPTIME UPDATE</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={
+            limit: 3,
+            sectionAlias: "videos/uptime-update",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">TOOL REVIEWS</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content",
+          params={
+            limit: 3,
+            sectionAlias: "videos/tool-reviews",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">KOLMAN'S CORNER</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content",
+          params={
+            limit: 3,
+            sectionAlias: "videos/kolmans-korner",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+    <div class="row">
+      <div class="col">
+        <h1 class="text-primary">TOOLBOX TOPICS</h1>
+        <marko-web-query|{ nodes }|
+          name="website-scheduled-content"
+          params={
+            limit: 3,
+            sectionAlias: "videos/toolbox-topics",
+            queryFragment }
+        >
+          <website-content-card-deck-flow nodes=nodes display-ads=false/>
+        </marko-web-query>
+      </div>
+    </div>
+
+    <hr>
+
+  </@page>
+  <@below-page>
+    <marko-web-load-more
+      component-name="content-card-deck-flow"
+      fragment-name="content-list"
+      query-name="all-published-content"
+      query-params={ contentTypes: ["Video"], limit: 10, skip: 5 }
```

Comment:
Should this page have load more?

Reply:
They asked for it to match the legacy page, and the legacy page had the load more.